ui.tests/test-module/specs/tabsontop/tabsontop.runtime.spec.js

```diff
@@ -208,4 +208,37 @@ describe("Form with TabsOnTop Layout Container with Hidden Children", () => {
             cy.get(`#${thirdItemOfTabId}`).should('have.class', 'cmp-tabs__tabpanel--active');
         })
     });
+});
+
+describe("setFocus of tabs using rules", () => {
+
+  const pagePath = "content/forms/af/core-components-it/samples/tabsontop/focusWithRule.html";
+  let formContainer = null;
+  beforeEach(() => {
+      cy.previewForm(pagePath).then(p => {
+          formContainer = p;
+      })
+  });
+
+  const tabSelector = 'ol li';
```

Comment:
Tab selector should refer BEM class right. Can you change it

Reply:
done